src/ripple/app/ledger/LedgerProposal.cpp

```diff
@@ -59,15 +60,12 @@ LedgerProposal::LedgerProposal (uint256 const& prevLgr, uint256 const& position,
 
 uint256 LedgerProposal::getSigningHash () const
 {
-    Serializer s ((32 + 32 + 32 + 256 + 256) / 8);
-
-    s.add32 (HashPrefix::proposal);
-    s.add32 (mProposeSeq);
-    s.add32 (mCloseTime);
-    s.add256 (mPreviousLedger);
-    s.add256 (mCurrentHash);
-
-    return s.getSHA512Half ();
+    return sha512Half(
+        HashPrefix::proposal,
+        std::uint32_t(mProposeSeq),
+        std::uint32_t(mCloseTime),
```

Comment:
Same as above. `mProposeSeq` and `mCloseTime` are already that type, so why cast?